task-native/src/main/java/org/ballerinalang/stdlib/task/utils/Utils.java

```diff
@@ -117,7 +119,8 @@ public static void validateService(ServiceInformation serviceInformation) throws
     private static void validateOnTriggerResource(BType returnParameterType) throws SchedulingException {
         if (returnParameterType != org.ballerinalang.jvm.types.BTypes.typeNull) {
             throw new SchedulingException(
-                    "Invalid resource function signature: \'" + TaskConstants.RESOURCE_ON_TRIGGER + "\' should not return a value.");
+                    "Invalid resource function signature: \'" + TaskConstants.RESOURCE_ON_TRIGGER +
+                            "\' should not return a value.");
```

Comment:
Shall we combine these three lines to two lines?

Reply:
Fixed in 5ef1e49